website/docs/sdk-reference/cpp.mdx

```diff
@@ -777,6 +777,32 @@ Info level logging helps to inspect how a feature flag was evaluated:
   Returning 'true'.
 ```
 
+### Custom logger implementation
+
```

Comment:
Could you add a short introduction text here like [the JS SDK docs do](https://github.com/configcat/docs/blob/fdf750f005a9ccb6996813146bf4ad88552cf404/website/docs/sdk-reference/js.mdx?plain=1#L618)?

Reply:
I moved the text from the parent section here.